src/confluent_kafka/src/Admin.c

```diff
@@ -454,19 +455,19 @@ static PyObject *Admin_create_topics (Handle *self, PyObject *args,
                 }
 
                 if (newt->replica_assignment) {
-                        if (newt->replication_factor != -1) {
+                        if (newt->replication_factor != -1 || newt->num_partitions !=-1) {
```

Comment:
space after the `!=`. i suspect you need to run `flake8`

Reply:
`flake8` doesn't work for C code.